.circleci/config.yml

```diff
@@ -270,6 +277,45 @@ jobs:
             - kernel/static
             - kernel/dist
 
+  build-website:
+    <<: *base_website_env
+    steps:
+      - <<: *attach_root
+      - run:
+          name: "installing"
+          command: npm ci
+      - run:
+          name: "remove decentraland-kernel"
+          command: rm -rf node_modules/decentraland-kernel || true
+      - run:
+          name: "copy fresh decentraland-kernel"
+          command: cp -r /tmp/explorer/kernel/static node_modules/decentraland-kernel
+      - run:
+          name: "post install script"
+          command: sh scripts/post-install.sh
+      - run:
+          name: "creating .env (ignore master)"
+          command: echo "PUBLIC_URL=\"https://play.decentraland.zone/branch-website/${CIRCLE_BRANCH}\"" >> .env
```

Comment:
This won't work in .today or .org

Reply:
@menduz Do you know how we can set the top level domain dynamically? can we know it from the branch name?

Reply:
we have no way to do it at any build stage of building currently, as they are branch agnostic... is there any chance to not require the public_url to generate the artifacts?

Reply:
it's require only for branches because we are deploying on a sub directory so we can use absolute paths. It's the reason why I ignore `master` and `tags` on lines 301 to 303